### ISSUE TITLE
Improve project documentation

### ISSUE THREAD

**Issue by ivalaginja** [lorenaacuna/GASTLI#4]
These are not strictly necessary for https://github.com/openjournals/joss-reviews/issues/7288 in this form, but addressing them would significantly improve your docs.

- I would separate out parts like Installation, Test suite and Community guidelines into a separate section from things like the part about the Atmospheric grid and the tutorials (linking to all of them from the welcome page). Right now it’s a bit confusing to understand the conceptual distinction between the sections.
- In the Quickstart section, it would help to have a bulleted overview of the individual steps that are covered in the quickstart part, so that when I open this page I know what I am signing up for and what the goals are. Maybe a table of contents? You have most of it in the intro paragraph of this doc page so it’s almost all there, just a little hard to parse when trying to find a first example to test the installation (although the automated test suite is there for that too).
- Your API documentation is great overall, however it would be even greater if your main initialization class, `GASTLI.int_planet`, would have complete docstrings, since it is the overarching class of the package. There should at least be a general docstring describing the class, and if you want to/can go all out, it would be great if its attributes also had docstrings.
- You have the required statement of need in your documentation (first paragraph on welcome page) but it is very minimal. I recommend extending it with a condensed version of what you wrote for the paper, in particular to address (a) what problems the software is designed to solve and (b) who the target audience is.

**Comment by lorenaacuna** [comment]
Thanks for the suggestions @ivalaginja. To address point 1), I grouped all the sub-sections (i.e "Installation", "Test suite", "Contributing", "Quickstart", etc) under a broader section titled "User Guide". Consequently, I removed the previous distinction between "User Guide" and "Tutorials". All of these sub-sections now have links in the sidebar and on the welcome page. I hope it is more clear now.

**Comment by lorenaacuna** [comment]
Comment 2): I added a table of contents to the Quickstart section in the documentation

**Comment by lorenaacuna** [comment]
Comment 3): There was a doctoring for this class, but it was placed after the `def __init__(...)` line, so it wasn't showing. Now it is corrected and it shows in the API's automatically generated documentation.

**Comment by lorenaacuna** [comment]
Comment 4): extended the text in the welcome page with target audience and a list of key features.

**Comment by ivalaginja** [comment]
I like it a lot better, thank you for addressing this!
